oneview_redfish_toolkit/blueprints/subscription_collection.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import logging
+
+
+from flask import abort
+from flask import Blueprint
+from flask import Response
+from flask_api import status
+
+from oneview_redfish_toolkit.api.subscription_collection \
+    import SubscriptionCollection
+
+subscription_collection = Blueprint("subscription_collection", __name__)
+
+
+@subscription_collection.route(
+    "/redfish/v1/SessionService/EventSubscriptions/", methods=["GET"])
+def get_subscription_collection():
+    """Get the Redfish Subscription Collection.
+
+        Get method to return SubscriptionCollection JSON when
+        /redfish/v1/SessionService/EventSubscriptions is requested.
+        Returns:
+                JSON: JSON with EventSubscriptions.
+    """
+    try:
+        # Build Subscription Collection object and validates it
+        sc = SubscriptionCollection()
```

Comment:
that dictionary would be passed here...

Reply:
But, if the dicts are in `util.py` I can retrieve them directly at `SubscriptionCollection` class. Is it better to pass them there or catch it directly in the class?

Reply:
Done as you said! :)